bucket/q-dir.json

```diff
@@ -38,12 +38,16 @@
     "extract_dir": "Q-Dir",
     "persist": [
         "Favoriten",
-        "Q-Dir.ini"
+        "Q-Dir.ini",
+        "start.qdr"
     ],
```

Comment:
```suggestion
    "pre_install": "if (!(Test-Path \"$persist_dir\\start.qdr\")) { New-Item \"$dir\\start.qdr\" | Out-Null }",
    "persist": [
        "Favoriten",
        "start.qdr",
        "Q-Dir.ini"
    ],
```

Reply:
shouldn't it be `$persist_dir` in the `New-Item \"$dir\\start.qdr\"` ?

Reply:
No.